src/components/tabs/Tabs.js

```diff
@@ -40,7 +40,7 @@ export default class TabsComponent extends NestedComponent {
     const components = _.cloneDeep(this.component.components);
     schema.components = components.map((tab, index) => {
       if (index === this.currentTab) {
```

Comment:
I am not sure if this if statement is needed anymore. It seems that the tabs would not update if you make multiple changes to multiple tabs.